src/commands/createWebApp/setDefaultRgAndPlanName.ts

```diff
@@ -12,28 +12,32 @@ import { createAzureClient, DialogResponses, IActionContext } from "vscode-azure
 import { ext } from "../../extensionVariables";
 import { nonNullProp } from "../../utils/nonNull";
 import { getWorkspaceSetting, updateGlobalSetting } from "../../vsCodeConfig/settings";
+import { AzConfig, AzConfigProperties, readAzConfig } from "./readAzConfig";
 
 const maxNumberOfSites: number = 3;
 
 export async function setDefaultRgAndPlanName(wizardContext: IAppServiceWizardContext, siteNameStep: SiteNameStep): Promise<void> {
+    const config: AzConfig = await readAzConfig(wizardContext, AzConfigProperties.group, AzConfigProperties.plan);
+
     // this should always be set when in the basic creation scenario
     const location: Location = nonNullProp(wizardContext, 'location');
-    const defaultName: string = `appsvc_${wizardContext.newSiteOS}_${location.name}`;
+    const defaultGroupName: string = config.group ? config.group : `appsvc_${wizardContext.newSiteOS}_${location.name}`;
```

Comment:
So if I have a default plan set:
```
> az configure --list-defaults --output json
[
  {
    "name": "plan",
    "source": "/Users/erijiz/.azure/config",
    "value": "emjtest1"
  }
]
```

I still see this when I try to create a webapp:
```
> az webapp create --name emjtest123 --resource-group emjtest123
az webapp create: error: the following arguments are required: --plan/-p
```

It seems like the cli doesn't use that default when creating? Can you double check the az cli's behavior and make sure you match it in terms of new vs. existing resources

Reply:
So plans aren't supported in the docs but the CLI lets you configure the plan anyway. We can remove support for checking plans if it's going to cause issues with commands like `list-defaults`

Reply:
I mean "--list-defaults" works just fine (it listed the plan). The problem is it wasn't actually used (from my brief testing) when I ran `az webapp create`

Reply:
Yeah default plans aren't formally supported. I can remove that